### DISCUSSION TITLE
Third party on hotmail. Hi, can anyone tell me what this is? If it was remotely put on or by someone who had access to my Hotmail?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#129688] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

![Screenshot_20240405_142740](https://github.com/community/community/assets/91843043/1f6687eb-7fb4-4c63-bf08-c617be976bcd)
![Screenshot_20240405_142626](https://github.com/community/community/assets/91843043/f86a43d4-2887-454f-bec0-d44d6aa8afd4)

![Screenshot_20240405_143223](https://github.com/community/community/assets/91843043/b681de94-293a-440c-bf3a-e1f9fe6eb2f5)
![Screenshot_20240405_143200](https://github.com/community/community/assets/91843043/c4a919ef-1139-45aa-9b06-e8d712ae4353)
![Screenshot_20240405_142853](https://github.com/community/community/assets/91843043/1025f68c-3709-4f1e-a7f9-23ce4ad413d5)

![Screenshot_20240419_192656](https://github.com/community/community/assets/91843043/0c04011c-cd12-46b8-b281-e671701be6f7)
Hi, can anyone tell me what this is? If it was remotely put on or by someone who had access to my Hotmail? 
![Screenshot_20240625_135020_Gallery](https://github.com/community/community/assets/91843043/0b4cf4b6-f2b1-4398-8dd7-7c51e1684551)

**Comment by LiteBrite82** [comment]
Hi @Ladyjuan ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
